staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -62,6 +66,11 @@ func TestOnlySetFatalOnDecodeError(b bool) {
 	fatalOnDecodeError = b
 }
 
+// testOnlySetDefaultWatcherMaxLimit should only be used for test case that paginate initial list for watch from zero.
+func testOnlySetDefaultWatcherMaxLimit(limit int64) {
```

Comment:
This function is not required because the test are defined in the same package. Feel free to remove it and directly manipulate `defaultWatcherMaxLimit`.

Reply:
+1